Makefile

```diff
@@ -1,81 +1,13 @@
 NAME    := fuse
+DEB_NAME := fuse3
 SRC_EXT := gz
 SOURCE   = https://github.com/libfuse/libfuse/archive/$(NAME)-$(VERSION).tar.$(SRC_EXT)
+ID_LIKE := $(shell . /etc/os-release; echo $$ID_LIKE)
+ifneq ($(ID_LIKE),debian)
 PATCHES := $(NAME).conf $(NAME)-install-nonroot.patch $(NAME)-linux-ioctl.patch
-
-COMMON_RPM_ARGS := --define "%_topdir $$PWD/_topdir"
-DIST    := $(shell rpm $(COMMON_RPM_ARGS) --eval %{?dist})
-ifeq ($(DIST),)
-SED_EXPR := 1p
 else
-SED_EXPR := 1s/$(DIST)//p
+PATCHES := $(NAME)-install-nonroot.patch $(NAME)-linux-ioctl.patch
```

Comment:
Rather than duplicating the list of patches for each branch, why not start with this list for all platforms and then add the `${NAME}.conf` in the `ifneq ($(ID_LIKE),debian)` branch?

Reply:
Will update